app/utils/SystemPropertiesHelper.scala

```diff
@@ -62,7 +62,7 @@ trait SystemPropertiesHelper {
     }
 
   def doesntExist[T](key: String, defaultvalue: T): T = {
-    Logger.warn(
+    Logger.debug(
```

Comment:
Hmm, I think I actually prefer this one at warn, we don't see debug logs anywhere but locally, but I'd want to know if we had missing config (which should really not happen).

Reply:
Reverted & pushed up.